pkgs/tools/misc/expect/default.nix

```diff
@@ -30,11 +31,16 @@ stdenv.mkDerivation {
   configureFlags = [
     "--with-tcl=${tcl}/lib"
     "--with-tclinclude=${tcl}/include"
-    "--exec-prefix=$out"
+    "--exec-prefix=$(out)"
   ];
 
-  postInstall = let libSuff = if stdenv.isDarwin then "dylib" else "so";
-    in "cp expect $out/bin; mkdir -p $out/lib; cp *.${libSuff} $out/lib";
+  postInstall = let libSuff = if stdenv.isDarwin then "dylib" else "so"; in ''
+    for i in $out/bin/*; do
```

Comment:
Oh, you're right. I've fixed `make install`, so the copying should be unneeded now.